tasks/lint.yaml

```
@@ -159,56 +159,92 @@ tasks:
   - name: license
     description: Lint for the SPDX license identifier being in source files
     actions:
-      - description: Checking for SPDX headers
+      - description: Checking for SPDX headers using YAML config with defaults
         shell:
           darwin: bash
           linux: bash
         cmd: |
-          # check for addlicense bin
-          if [ -x "$HOME/go/bin/addlicense" ]; then
-              echo "addlicense installed in $HOME/go/bin"
+          CONFIG_FILE="license_config.yaml"
```

did you want this to be `license_config.yaml`? or `.license_config.yaml`?

```suggestion
          CONFIG_FILE=".license_config.yaml"
```